app/sign.js

```diff
@@ -18,6 +18,8 @@ const utxoNetworks = {
   tbch: utxoLib.networks.bitcoincashTestnet,
   tzec: utxoLib.networks.zcashTest,
   tdash: utxoLib.networks.dashTest,
+  btg: utxoLib.networks.bitcoingold,
+  tbtg: utxoLib.networks.bitcoingold
 };
```

Comment:
i don't think we even have a tbtg on the platform, but in case we ever add it (doubtful), we'll be ready